pjsip/src/pjsua-lib/pjsua_vid.c

```diff
@@ -177,7 +177,12 @@ pj_status_t pjsua_vid_subsys_destroy(void)
         }
     }
 
-    /* Destroy avi file players */
```

Comment:
For future ref, better put comment about why the vid-conf need to be destroyed before AVI player, etc?